code/game/machinery/suit_storage_unit_vr.dm

```diff
@@ -1,8 +1,9 @@
 // Old Exploration is too WIP to use right now
+/*
```

Comment:
Exploration voidsuits are not going away, these should be kept, just moved into the gateway or something.